index.css

```diff
@@ -384,3 +384,88 @@ button:disabled {
   align-items: center;
   gap: 15px;
 }
+
+/* Contact form */
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  padding: 120px 24px 0 24px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact .content {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  margin-bottom: 50px;
+  line-height: 44px;
+  text-align: center;
+  align-content: center;
+  color: #172b4d;
+}
+
+input,
+textarea {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  width: 356px;
+  border: none;
+  padding-left: 5px;
+  margin-top: 25px;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+}
```

Comment:
- [ ] Kindly make the `width: 356px;` smaller, because it makes an overflow in your page.

![image](https://user-images.githubusercontent.com/35535130/155238327-3a46c320-9acd-4dda-a664-87d9fac054e9.png)